src/components/DayPicker.jsx

```diff
@@ -28,6 +28,7 @@ const propTypes = {
   withPortal: PropTypes.bool,
   hidden: PropTypes.bool,
   initialVisibleMonth: PropTypes.func,
+  scrollable: PropTypes.bool,
```

Comment:
So this prop is just going to be a clusterfuck if we happen to have the orientation set to horizontal, right? `airbnb-prop-types` has some shit about exclusive boolean props, but that's not quite what's happening here. Could we create a rule for this?